PSKoans/Public/Show-Karma.ps1

```diff
@@ -80,6 +85,29 @@ function Show-Karma {
                 $KoanLocation | Invoke-Item
             }
         }
+        'OpenFile' {
+            try {
+                $Results = Get-Karma @GetParams
+            }
+            catch {
+                $PSCmdlet.ThrowTerminatingError($_)
+            }
+            $FilePath = Get-PSKoan -Topic $Results.CurrentTopic.Name -Scope User | Select-Object -ExpandProperty Path
+            $LineNumber = $Results.CurrentTopic.FailedLineNumber
+            if ($PSVersionTable.Platform -eq "win32nt") {
+                $Editor = (Get-ItemProperty -Path 'HKCU:\Software\Microsoft\Windows\CurrentVersion\Explorer\FileExts\.ps1\UserChoice').ProgId
+                $Editor =  [regex]::matches($DefaultProgram,'(?<=\\).+(?=.exe)').value
+                switch ($Editor) {
+                    {($_ -eq "code") -or ($_ -eq "code-insiders")} {$FullArgument = '-g "{0}":{1}' -f $FilePath, $LineNumber; Start-Process $Editor -ArgumentList $FullArgument}
+                    atom {$FullArgument = '"{0}":{1}' -f $FilePath, $LineNumber; Start-Process $Editor -ArgumentList $FullArgument}
+                    powershell_ise {$FullArgument = "{0}" -f $FilePath; Start-Process $Editor -ArgumentList $FullArgument}
```

Comment:
```suggestion
                    {$_ -in 'powershell_ise', 'isep' } {
                        $FullArgument = "{0}" -f $FilePath
                        Start-Process -FilePath $Editor -ArgumentList $FullArgument
                    }
```

Should probably cover the ISE-preview editor here as well. Probably not a lot of folks using it, but it doesn't hurt to be careful.